code/game/objects/effects/temporary_visuals/clockcult.dm

```diff
@@ -153,53 +153,6 @@
 /obj/effect/temp_visual/ratvar/component/ansible
```

Comment:
These visual should be moved somewhere else

Reply:
Where to though, I could just rename the file to clockwork.dm since they are clockwork visuals rather than clockcult visuals. They are still used by the brass walls however